_health-care/_js/_components/_hello-world.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { IndexLink, Link } from 'react-router'
+
+import DateInput from './date_input';
+
+class Nav extends React.Component {
+  render() {
+    return (
+      <div>
+        <Link to="/hello">Hello</Link><br/>
+        <Link to="/night">Night</Link><br/>
+        <IndexLink to="/">Home</IndexLink>
+      </div>
+    )
+  }
+}
+
+class Hello extends React.Component {
+  render() {
+    return(
+      <div>
+        <h2>Hello World</h2>
+
+        <div className="row">
+          <h3>Happy Date</h3>
+          <DateInput date={this.props.applicationData.happyDate}
```

Comment:
Eventually, when `Hello` is actually a panel or block of questions (some intermediary component), should it have access to all of `applicationData`? Is there a way for each layer to only know about the data for the layers it contains? Maybe `applicationData` has nested objects within it that correspond to different layers? This is just thinking in advance